RInAppMessaging/Classes/UserDataCache.swift

```diff
@@ -74,7 +74,16 @@ internal class UserDataCache: UserDataCacheable {
         }
     }
 
-    private func userKey(from identifiers: [UserIdentifier]) -> Set<UserIdentifier> {
-        Set<UserIdentifier>(identifiers)
+    private func userKey(from identifiers: [UserIdentifier]) -> String {
+        var hasher = KeyHasher()
+        identifiers.map({ $0.identifier }).sorted().forEach {
+            hasher.combine($0)
+        }
+        hasher.encryprtionMethod = .md5
+        let salt = hasher.generateHash()
```

Comment:
So the salt is the md5 hash of the user ids?

Reply:
Yes. I'm not sure if it makes key more secure but it's something 🙂